hil-test/tests/sha.rs

```diff
@@ -216,6 +216,7 @@ mod tests {
     /// A test that runs a hashing on a digest of every size between 1 and 200
     /// inclusively.
     #[test]
+    #[timeout(15)]
```

Comment:
given the previous overall default was 10s, 15s seem to be quite long - but I don't think it really matters

Reply:
With 10, the S2 failed here.

Reply:
ah sorry - the previous overall-default was 60s 👀